scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -2903,7 +2903,8 @@ def _build_fetches(self, global_step, all_logits, losses, device_grads,
         key = name[len(constants.UNREDUCED_ACCURACY_OP_PREFIX):]
         fetches[key] = tf.concat(ops, 0)
       else:
-        fetches[name] = tf.reduce_sum(ops) / self.batch_size
+        fetches[name] = (tf.reduce_sum(ops) / self.batch_size *
+                         (self.num_workers if self.single_session else 1))
```

Comment:
I think you want to divide by num_workers here, not multiply. Or alternatively, put parenthesis around `self.batch_size * (self.num_workers if self.single_session else 1)`

Reply:
You're right. I want to divide num_workers if single_session is used. Your alternative suggestion looks better. 👍

Reply:
I've updated it.

Best regards,
Gyeongchan Yun